test_schemas/en/test_question_guidance.json

```diff
@@ -27,7 +27,7 @@
             "groups": [
                 {
                     "id": "group",
-                    "title": "",
+                    "title": "Group",
```

Comment:
All other places where a generic group title is needed use "Title" rather than "Group". This should be changed to match so that it's easy to find and remove when we sort out the group titles issue.